packages/pagebuilder/lib/ContentTypes/Slider/slider.js

```diff
@@ -88,7 +89,7 @@ const Slider = props => {
         autoplaySpeed,
         fade
     };
-
+    sliderSettings.dots = false;
```

Comment:
Instead of changing the dots setting here, please change the default values in
`packages/pagebuilder/lib/ContentTypes/Slider/configAggregator.js` to:

```
showArrows: node.getAttribute('data-show-arrows') === 'false',
showDots: node.getAttribute('data-show-dots') === 'false',
```